proxy/nginx/lualib/router.lua

```diff
@@ -1,4 +1,3 @@
-local container_url = ngx.var.container_url
```

Comment:
Could this be causing https://github.com/sourcelair/ceryx/pull/39#issuecomment-358246793?